bindings/python/client.py

```diff
@@ -110,11 +110,12 @@ def print_peer_info(console, peers):
         else: out += 'r'
         out += ' '
 
-        if p.downloading_piece_index >= 0:
-            assert(p.downloading_progress <= p.downloading_total)
-            out += progress_bar(float(p.downloading_progress) / p.downloading_total, 15)
-        else:
-            out += progress_bar(0, 15)
+        # TODO TypeError: No Python class registered for C++ class libtorrent::aux::strong_typedef<int, libtorrent::aux::piece_index_tag, void>
```

Comment:
ah, I think I know what the problem is. the ``downloading_piece_index`` member need to be bound as a property, to be able to be convertible with custom converters.